pkg/provision/workspace/deployment.go

```diff
@@ -121,7 +121,7 @@ func SyncDeploymentToCluster(
 	}
 
 	if clusterDeployment == nil {
-		clusterAPI.Logger.Info("Creating deployment...")
+		clusterAPI.Logger.Info("Creating deployment")
```

Comment:
deployment name can be useful here as well, but since ATM we have only one and one deployment, that's OK